src/async_fuse/memfs/cache/memory_cache/cache.rs

```diff
@@ -306,6 +306,18 @@ where
     }
 
     async fn remove(&self, ino: INum) {
+        if !self.write_through {
+            let mut pending_write_back = self.pending_write_back.write().await;
+            if self
+                .command_sender
+                .send(Command::Cancel(ino))
+                .await
+                .is_err()
+            {
+                warn!("Write back task is closed unexpectedly.");
```

Comment:
Is warning enough, shall we fallback to the write through policy here?

Reply:
It's enough. This is to cancel the flushing of the file, if the write back task is closed, these blocks will not be flushed to the backend anymore.

Reply:
But I will double check the related codes in `MemoryCache`, to ensure that the block to be flushed will not be ignored when the write back task is closed unexpectedly.